src/main/resources/META-INF/resources/lib/index.es.js

```diff
@@ -14,32 +14,47 @@
 
 import ClayButton from '@clayui/button';
 import ClayCard from '@clayui/card';
+import {ClayToggle} from '@clayui/form';
 import ClayProgressBar from '@clayui/progress-bar';
-import {openSelectionModal} from 'frontend-js-web';
+import {createPortletURL, openSelectionModal} from 'frontend-js-web';
 import React, {Fragment, useState} from 'react';
 import ReactDOM from 'react-dom';
 
 const Pokemon = ({eventName, itemSelectorURL}) => {
-	const [selectedItem, setSelectedItem] = useState();
+	const [selectedItems, setSelectedItems] = useState([]);
+	const [multiple, setMultiple] = useState(false);
 
 	return (
 		<Fragment>
 			<ClayButton
 				onClick={() =>
 					openSelectionModal({
 						onSelect: ({value}) =>
-							setSelectedItem(JSON.parse(value)),
+							setSelectedItems(
+								multiple
+									? value.map((item) => JSON.parse(item))
+									: [JSON.parse(value)]
+							),
 						selectEventName: eventName,
 						title: Liferay.Language.get('select-a-pokemon'),
-						url: itemSelectorURL,
+						url: createPortletURL(itemSelectorURL, {
```

Comment:
This won't be needed when this [pull](https://github.com/liferay-frontend/liferay-portal/pull/392) gets merged in master. So this will be simplified to just `url:itemSelectorURL`